about.html

```diff
@@ -8,57 +8,102 @@
 
   <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
   <link rel="stylesheet" href="css/index.css">
+  <link rel="stylesheet" href="css/about.css">
 
 </head>
 
-<body>
-    <div class="container about-page">
-     
-    About
+<div id="headerWrap">
```

Comment:
I strongly advice keeping all your <divs> inside the <body>
Also consider using auto-formatting options inside your IDE. Here is a VS code tutorial: https://zellwk.com/blog/vscode-2/

Reply:
Thanks for the feedback very much appreciate it. I will take a look at that link you sent me.